packages/wonder-blocks-icon-button/src/components/node-icon-button.tsx

```diff
@@ -0,0 +1,315 @@
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+import {Link} from "react-router-dom-v5-compat";
+
+import {StyleType, View} from "@khanacademy/wonder-blocks-core";
+import {PhosphorIcon} from "@khanacademy/wonder-blocks-icon";
+import {focusStyles} from "@khanacademy/wonder-blocks-styles";
+import {border, semanticColor, sizing} from "@khanacademy/wonder-blocks-tokens";
+
+import type {BaseIconButtonProps} from "../util/icon-button.types";
+
+import {IconButtonUnstyled} from "./icon-button-unstyled";
+
+/**
+ * The object containing the CSS variables that can be overridden to customize
+ * the appearance of the NodeIconButton component.
+ */
+type Tokens = Partial<{
+    "--wb-c-node-icon-button--box-foreground": string;
+    "--wb-c-node-icon-button--box-background": string;
+    "--wb-c-node-icon-button--box-shadow-color": string;
+    "--wb-c-node-icon-button--box-padding": string | number;
+    "--wb-c-node-icon-button--box-shadow-y-rest": string | number;
+    "--wb-c-node-icon-button--box-shadow-y-hover": string | number;
+    "--wb-c-node-icon-button--box-shadow-y-press": string | number;
+    "--wb-c-node-icon-button--icon-size": string | number;
+}>;
+
+type Props = Omit<BaseIconButtonProps, "kind" | "style"> & {
+    /**
+     * The action type of the button. This determines the visual style of
+     * the button. Defaults to `notStarted`.
+     *
+     * - `notStarted` is used for buttons that indicate a not started action.
+     * - `attempted` is used for buttons that indicate an attempted (in progress)
+     *   action.
+     * - `complete` is used for buttons that indicate a complete action.
+     */
+    actionType?: "notStarted" | "attempted" | "complete";
+    /**
+     * The alternative text for the icon button. Required for accessibility.
+     */
+    "aria-label": string;
+
+    /**
+     * The size of the icon button.
+     * One of `small` (24) or `large` (68).
+     * Defaults to `large`.
+     */
+    size?: "small" | "large";
+
+    /**
+     * Custom styles for the elements in the NodeIconButton component.
+     * - `root`: Styles the root element (button)
+     * - `box`: Styles the "chonky" box element
+     * - `icon`: Styles the icon element
+     */
+    styles?: {
+        root?: StyleType;
+        box?: StyleType;
+        icon?: StyleType;
+    };
+};
+
+/**
+ * Node buttons are visual representations of activities along in a Learning
+ * Path. When a represented No  de is a button that launches the activity. Nodes
```

Comment:
```suggestion
 * Path. When a represented Node is a button that launches the activity. Nodes
```

Reply:
Forgot to mention, but I fixed this in the second PR (tokens pr).